docusaurus.config.js

```diff
@@ -1,5 +1,5 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
```

Comment:
See my comment in https://github.com/Seneca-ICTOER/IPC144/pull/60#discussion_r747628232 and fix please.

Reply:
I can talk with @dbelokon so I can address this in a separate PR.